pom.xml

```diff
@@ -45,40 +45,37 @@
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>appassembler-maven-plugin</artifactId>
         <version>2.0.0</version>
-        <configuration>
-          <extraJvmArguments>-Xms512M -Xmx24576M</extraJvmArguments>
-          <programs>
-            <program>
-              <mainClass>io.anserini.index.IndexCollection</mainClass>
-              <name>IndexCollection</name>
-            </program>
-            <program>
-              <mainClass>io.anserini.search.SearchCollection</mainClass>
-              <name>SearchCollection</name>
-            </program>
-          </programs>
-        </configuration>
-      </plugin>
-      <plugin>
-        <artifactId>maven-assembly-plugin</artifactId>
-        <configuration>
-          <archive>
-            <manifest>
-              <mainClass>fully.qualified.MainClass</mainClass>
-            </manifest>
-          </archive>
-          <descriptorRefs>
-            <descriptorRef>jar-with-dependencies</descriptorRef>
-          </descriptorRefs>
-        </configuration>
+        <executions>
+          <execution>
+            <phase>package</phase>
+            <goals>
+              <goal>assemble</goal>
+	    </goals>
+            <configuration>
+              <extraJvmArguments>-Xms512M -Xmx24576M</extraJvmArguments>
+              <programs>
+                <program>
+                  <mainClass>io.anserini.index.IndexCollection</mainClass>
+                  <id>IndexCollection</id>
+                </program>
+                <program>
+                  <mainClass>io.anserini.search.SearchCollection</mainClass>
+                  <id>SearchCollection</id>
```

Comment:
I think you had accidentally removed `Eval` and `IndexUtils` in one recent PR?
Can you add them back?

Reply:
done